mlsdbtool/mcc_update.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# This will take the mcc data from the file "data/MCC" in the repository
+# tzdata-timed ( https://github.com/sailfishos/tzdata-timed ) and use it
+# to update the code in this repository.
+
+declare -i mcc
+HEADER="../common/mccmapping.h"
+SCRIPT="./mccmapping"
+mcc_file=$1
+if [ -z "$mcc_file" ] ; then
+    echo "Usage: $0 [MCC file]" >&2
+    exit 1
+fi
+if [ ! -e $HEADER ] ; then
+    echo "ERROR: Unable to locate header $HEADER" >&2
+    exit 1
+fi
+if [ ! -e $SCRIPT ] ; then
+    echo "ERROR: Unable to locate script file $SCRIPT" >&2
+    exit 1
+fi
+
+mccs=`cat $mcc_file | grep -v ^# | cut -d ' ' -f1 | sort -nu | tr '\n' ',' | sed 's/^,//;s/,$/,1024/'`
+count=`echo -n $mccs | tr -dc ',' | wc -c`
```

Comment:
shellcheck'd:
```
count=`echo -n $mccs | tr -dc ',' | wc -c`
      ^-- SC2006: Use $(...) notation instead of legacy backticked `...`.
               ^---^ SC2086: Double quote to prevent globbing and word splitting.

Did you mean: 
count=$(echo -n "$mccs" | tr -dc ',' | wc -c)